client/src/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import styled from 'styled-components';
+import Product from './components/Product.jsx';
+import Styles from './components/Styles.jsx';
+
+const ProductInfo = styled.div`
+  display: flex;
+  flex-flow: column;
+`;
+
+class ProductModule extends React.Component {
+
+  constructor(props) {
+    const ids = window.location.pathname.split('/');
+    super(props);
+    this.state = {
+      productId: ids[2],
+      styleId: ids[3],
+      name: '',
+      gender: '',
+      category: '',
+      style: {},
+      styles: [],
+    };
+  }
+
+  setThumbnails(styles, photos) {
+    let stylesUpdated = [];
+    styles.map(style => {
+      photos.map(photo => {
+        if (photo.style_id === style.styleId) {
+          style['imageThumbnail'] = photo.main_photo.thumbnail_url;
+          style['imageRegular'] = photo.main_photo.regular_url;
+          stylesUpdated.push(style);
+        }
+      });
+    });
+    return stylesUpdated;
+  }
+
+  componentDidMount() {
+    $.ajax({
+      url: `http://localhost:3008/product/${this.state.productId}`
```

Comment:
Change `localhost:3008` to a variable defined in an config file so when you deploy it's easy to set.
urls: {
dev: {
product: "localhost:3008"
},
 prod: {
product: "aws endpoint: 3008"
}
}